MUXSDKStats/MUXSDKStats/MUXSDKPlayerBinding.m

```diff
@@ -887,6 +887,12 @@ - (void)dispatchError {
 }
 
 - (void) dispatchError:(nonnull NSString *)code withMessage:(nonnull NSString *)message {
+    [self dispatchError:code withMessage:message withErrorContext:nil];
```

Comment:
What do you think about adding a simple test for this API to `MUXSDKPlayerBindingTests`?